unity-client/Assets/Batch/AssetBundleBuilder.cs

```diff
@@ -55,7 +56,7 @@ public enum ErrorCodes
 
         public Dictionary<string, string> hashLowercaseToHashProper = new Dictionary<string, string>();
 
-        internal ContentServerUtils.ApiEnvironment environment = ContentServerUtils.ApiEnvironment.ORG;
+        internal ContentServerUtils.ApiTLD tld = ContentServerUtils.ApiTLD.ORG;
```

Comment:
What does TLD stand for? Not a big fan of acronyms unless they are plain obvious

Reply:
TLD = Top level domain, the reason for the change is that same term is used for this purpose at kernel, so I thought it'd be better to improve terms ubiquity between both projects.